src/lib/components/consignments/__stories__/bottom_aligned.story.tsx

```diff
@@ -1,8 +1,8 @@
 import * as React from "react"
 import { Text, View } from "react-native"
 
-import Search from "../components/artist-search-results"
-import BottomAlignedButton, { BottomAlignedProps } from "../components/bottom-aligned-button"
+import Search from "../components/artist_search_results"
+import BottomAlignedButton, { BottomAlignedProps } from "../components/bottom_aligned_button"
```

Comment:
bunch of renames, they're in a folder called consignments already, so no need for extra context